### PR TITLE
Added tag "Meat" to Brain Cake and Brain Cake Slice

### PR DESCRIPTION
**Pull request by Catofquestionableethics**
## About the PR
Added Tag "Meat and Cake" to brain cake, so Lizard Species can eat it.

## Why / Balance
Allows Lizards to eat brain cake, which they could not previously.

## Technical details
Added Tag "Meat and Cake" to Brain Cake and "Meat, Cake and Slice" to it's slices

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- fix: Brain Cake is now meat, and can be eaten by Lizards.

### CODE REVIEW

**Comment by Golinth** [comment]
any chance you could add it to meatzu while here? You can normally eat it but lizards cannot because it doesn't have the meat tag.

**Comment by Catofquestionableethics** [comment]
> any chance you could add it to meatzu while here? You can normally eat it but lizards cannot because it doesn't have the meat tag.

I feel like that would fall under a seperate pr? I dont know.

**Comment by Golinth** [comment]
All good, you don’t need to, and it probably would be best to put it under a separate PR

**Comment by TheShuEd** [comment]
> > any chance you could add it to meatzu while here? You can normally eat it but lizards cannot because it doesn't have the meat tag.
> 
> I feel like that would fall under a seperate pr? I dont know.

feel free to fix it here

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 1]
```diff
@@ -168,7 +168,9 @@
     state: brain
   - type: SliceableFood
     slice: FoodCakeBrainSlice
-
+  - type: Tag
+    tags:
+    - Meat
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:173]:
```suggestion
    - Meat
    - Cake
```
image

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 2]
```diff
@@ -178,6 +180,9 @@
   components:
   - type: Sprite
     state: brain-slice
+  - type: Tag
+    tags:
+    - Meat
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:185]: same here
```suggestion
    - Meat
    - Cake
    - Slice
```

**Review by TheShuEd — APPROVED** [review]
fix tags

**Comment by Catofquestionableethics** [comment]
I think that did it? Thanks Ed!

**Comment by Catofquestionableethics** [comment]
I have gone over the Cake.Yml file and it is full of inconsistencies. Would it be worth closing this PR and fixing every cake, or making a seperate PR to do that?

**Comment by Catofquestionableethics** [comment]
I've looked at the Cake.Yml file and there's a bunch of tweaks/changes that probably should be made. Its full of things like carrot cake not being labeled as Vegetable, Christmas Cake not being labelled fruit and Spacemans Cake Containing omni. Im gonna close this PR and make a second one with all the changes.

**Pull request closed without merging**